src/coreclr/vm/callingconvention.h

```diff
@@ -308,6 +308,14 @@ struct TransitionBlock
         {
             return argLocDescForStructInRegs->m_cFloatReg > 0;
         }
+    #elif defined(TARGET_LOONGARCH64)
+        if (argLocDescForStructInRegs != NULL)
+        {
+            if (argLocDescForStructInRegs->m_structFields & STRUCT_FLOAT_FIELD_SECOND)
+            {
+                return argLocDescForStructInRegs->m_cFloatReg > 0;
+            }
```

Comment:
Nit: Is that condition necessary? I think `argLocDescForStructInRegs->m_cFloatReg > 0` alone would suffice.

Reply:
Yes, this is somewhat conservative. It should be okay without this condition.

Reply:
Thanks.